mantidimaging/gui/widgets/mi_mini_image_view/view.py

```diff
@@ -72,15 +72,15 @@ def histogram(self) -> HistogramLUTItem:
 
     @property
     def histogram_region(self) -> tuple[int | list[int], int | list[int]]:
-        return self.hist.region.getRegion()
+        return self.hist.region.getRegion()  # type: ignore
 
     @histogram_region.setter
     def histogram_region(self, new_region: tuple[int | list[int], int | list[int]]) -> None:
         self.hist.region.setRegion(new_region)
 
     @property
     def image_data(self) -> np.ndarray:
-        return self.im.image
+        return self.im.image  # type: ignore
```

Comment:
We should change the signature to that callers have to handle the None case.

Reply:
Played around with this and have set the return to `np.ndarray | None` and added an assertion in the spectrum viewer which relied on `self.im.image` not being None. If I threw in an assert in `image_data` instead of changing the return type, We get a lot of errors due to there being instances where `self.im.image` is None.